vimeo/VimeoNetworking
# Project fixes for Carthage integration, Podspec Updates to include tests

#### Ticket

N/A - Fixes #328 

#### Pull Request Checklist


#### Issue Summary

This PR contains a couple regression fixes due to project file structure changes that happened on PR #327:

- Carthage integration was failing due to Pods/* directory being .gitignore'd so this change was reverted
- Fastfile project and scheme names have been changed to match the new structure

Unrelated to these changes, I have also taken the opportunity to include 
a test spec as part of our VimeoNetworking spec. To confirm that this is working as
intended you can run `pod lib lint` and ensure the linter execution is successful.

#### How to Test

- Run `carthage build --no-skip-current` and ensure it completes successfuly
- Run `pod lib lint` and ensure it completely successfuly.
